Turf/Turf.swift

```diff
@@ -285,3 +288,67 @@ public struct Polyline {
         return closestCoordinate
     }
 }
+
+public struct Polygon {
+    var polygonCoordinates: [[CLLocationCoordinate2D]]
+    
+    var polygonArea: Double {
```

Comment:
Add a comment with a permalink to the specific commit of the Turf package you’re porting this code from, in case we need to track down a discrepancy in the future.